### DISCUSSION TITLE
🎉Building a Custom Source Connector in Airbyte : LowCode CDK

### DISCUSSION THREAD

**Discussion by avirajsingh7** [airbytehq/airbyte#29332] [Show and tell]
Hello folks,
I am Aviraj Gour, I hope you are doing well. I recently published a blog post where I went deep into the process of building Custom Connector in airbyte. In the blog, I detailed a step-by-step guide on creating connectors without any coding experience.

**Blog Link** : https://aviraj-writes.hashnode.dev/airbyte-lowcode-connector

I'd love to hear your thoughts, questions, and experiences related to this topic.

Feel free to share:

- Any challenges you've faced while building connectors using LowCode CDK.
- Tips and tricks you've discovered along the way.
- Questions you have about the process described in the blog.

I hope through this discussion we can clear everyone doubts regarding LowCode CDK and mine too 😄

Looking forward to engage with you all. 🚀

**Comment by saulBurgosProtiv** [comment]
Thank you for the tutorial.

**Comment by mooSylla** [comment]
Is this up to date  cause I cant find the airbyte-integrations/connector-templates/generator
./generate.sh
